dandiapi/zarr/tests/test_zarr_upload.py

```diff
@@ -11,20 +11,20 @@
 
 @pytest.mark.django_db
 def test_zarr_rest_upload_start(
-    authenticated_api_client, user, zarr_archive: ZarrArchive, storage, monkeypatch
+    authenticated_api_client, user, embargoed_zarr_archive_factory, storage, monkeypatch
 ):
+    zarr_archive: ZarrArchive = embargoed_zarr_archive_factory.create(
```

Comment:
Beyond switching to a factory, this needs to be an *embargoed* Zarr, or the test won't pass. Only embargoed Zarr should actually have tags applied, as we assert.

Reply:
Obsoleted by #2503